scripts/helmcharts/openreplay/templates/job.yaml

```diff
@@ -119,6 +119,10 @@ spec:
             value: "{{ .Values.minio.global.minio.accessKey }}"
           - name: MINIO_SECRET_KEY
             value: "{{ .Values.minio.global.minio.secretKey }}"
+          - name: MINIO_HOST
+            value: "{{ .Values.minio.global.minio.minioHost }}"
+          - name: MINIO_POST
+            value: "{{ .Values.minio.global.minio.minioPort }}"
```

Comment:
we have to derive these values from global.s2.endpoint

Reply:
pushed an update regarding this. I switched it back to that `global.s3.endpoint`. I also update the if statement before that to check if it is a cluster dns and not a storage endpoint dns